app/src/main/kotlin/com/wire/android/util/DateTimeUtil.kt

```diff
@@ -181,10 +177,7 @@ sealed interface MessageDateTimeGroup {
 
 fun String.uiMessageDateTime(): String? = this
     .serverDate()?.let { serverDate ->
-        when (DateUtils.isToday(serverDate.time)) {
-            true -> messageTimeFormatter.format(serverDate)
-            false -> messageDateTimeFormatter.format(serverDate)
-        }
+        messageTimeFormatter.format(serverDate)
```

Comment:
What is the goal of this change? Not sure if this is related to the current PR description.

Reply:
Forgot to mention on the description, sorry :(

This change is for removing the date and only showing the time on the right side of the message item, as we are adding dividers with dates, there is no reason to have it on the message item as well.

based on the designs and followed up with our team designer